packages/contentstack-import/src/utils/content-type-helper.ts

```diff
@@ -90,7 +90,7 @@ export const removeReferenceFields = async function (
       for (var block in schema[i].blocks) {
         await removeReferenceFields(schema[i].blocks[block].schema, flag, stackAPIClient);
       }
-    } else if (schema[i].data_type === 'reference' || schema[i].reference_to) {
+    } else if (schema[i].data_type === 'reference') {
```

Comment:
@shafeeqd959  / @aman19K  Could someone verify here whether removing this condition will have no impact on functionality?

While the previous code didn't have it, the rewrite shows some changes. Seeking clarity before approving the PR to avoid new issues while addressing the current one.

cc @netrajpatel @abhinav-from-contentstack @cs-raj

Reply:
I think this condition may have been added as a check if `reference_to` field exists in the object, as `reference_to` field is being used in a loop.
https://github.com/contentstack/cli/blob/c005a1343aad6a16d041dc68327e52f1ee6df5e6/packages/contentstack-import/src/utils/content-type-helper.ts#L93-L106

The problem is that objects that have `data_type: 'reference'` and `data_type: 'json'` ( in case of Json Rte ), both have `reference_to` fields.

As the condition on line 93 is an OR condition, objects with `data_type: 'json'` also get passed. And that should not happen, as this block is specifically written to handle `data_type: 'reference'` objects.

Although I've never seen an object with `data_type: reference` that doesn't have a `reference_to` array, if we are skeptical then maybe replacing `||` with `&&` will also work... atleast in the context of the Prime Demo issue.

@cs-raj may check if this also works for his fix.
Let me know what you guys think.
cc : @aman19K @antonyagustine @shafeeqd959

Reply:
@abhinav-from-contentstack I tried the above it imported the global fields without issue from the selinc stack.
cc: @aman19K @antonyagustine @shafeeqd959